pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -1573,6 +1573,13 @@ boolean isUpdateable() throws SQLException {
       String[] s = quotelessTableName(tableName);
       String quotelessTableName = s[0];
       String quotelessSchemaName = s[1];
+      if (quotelessSchemaName.isEmpty() && !isTemporaryTable(quotelessTableName)) {
```

Comment:
Why are temporary tables excluded here?
Could you add a test case for temporary tables as well?